docs/topics/api/accounts.rst

```diff
@@ -2,64 +2,83 @@
 Accounts
 ========
 
-.. note:: This API requires :doc:`authentication <auth>`.
-
 The following API endpoints cover a users account.
 
-.. _`profile`:
 
 -------
 Profile
 -------
 
-Returns information about your profile.
-
-.. http:get:: /api/v3/accounts/profile/
-
-    **Request:**
+.. _`profile`:
 
-    .. sourcecode:: bash
+This endpoint returns information about a user's profile, by the profile id.
+Most of the information is optional and provided by the user so may be missing or inaccurate.
 
-        curl "https://addons.mozilla.org/api/v3/accounts/profile/"
-            -H "Authorization: JWT <jwt-token>"
+.. http:get:: /api/v3/accounts/profile/(int:user_id)/
 
-    **Response:**
+    .. _profile-object:
 
-    .. sourcecode:: json
+    :>json int id: The numeric user id.
+    :>json string username: username chosen by the user, used in the profile url. If not set will be a randomly generated string.
+    :>json string name: The name chosen by the user, or the username if not set.
+    :>json array addons_listed: Array holding the :ref:`addons <addon-detail-object>` the user has developed and are listed on this website.
+    :>json float averagerating: The average rating for the `addons_listed`.
```

Comment:
I vote for changing this field name in the API to at least `average_rating` - or something more self-descriptive.